src/main/java/com/commercetools/pingpong/model/Game.java

```diff
@@ -91,5 +126,67 @@ public Player getPlayerTwo() {
         return playerTwo;
     }
 
-}
+    public boolean getPlayerOneServe() {return playerOne.amIServing();}
+
+    public boolean getPlayerTwoServe() {return playerTwo.amIServing();}
+
+    public boolean isOvertime(Player playerOne, Player playerTwo) {
+        return playerOne.getSetScore() >= 10 && playerTwo.getSetScore() >= 10;
+    }
+
+    public void whoServesAfterDoubleClick() {
+        if (!isSetScoreEven()) {
+            if (playerOne.amIServing()) {
+                playerOne.changeServe();
+                playerTwo.setServe();
+
+            } else if (playerTwo.amIServing()) {
+                playerTwo.changeServe();
+                playerOne.setServe();
+            }
+        }
+    }
 
+    public void setFirstServe(Player actingPlayer) {
+        actingPlayer.setBeginningServe();
+        actingPlayer.setServe();
+        }
+
+
+    public void setBeginningServe() {
```

Comment:
Both the game and the player have a `setBeginningServe` method? 🤔

Reply:
Oh, that's right...
595f3e3389620dfe24617ce2375bb075f8b6d2b8